FilmRatings/FilmRatings/settings.py

```diff
@@ -29,6 +29,7 @@
 ALLOWED_HOSTS = [
     'localhost',
     'air-van-maarten',
+    '127.0.0.1',
```

Comment:
Please  disallow this IP address and use port numbers to distinguish old en new version.
As long as that is even necessary.

Reply:
Done.